tiledb/ml/models/cloud_utils.py

```diff
@@ -0,0 +1,28 @@
+import os
+import tiledb
+import tiledb.cloud
+
+from typing import Union
+
+from . import CLOUD_MODELS
+
+
+def get_s3_prefix(namespace: str) -> Union[str, None]:
```

Comment:
If it is `utils` as it seems to be and I think we start needing it more and more day by day let's consider create a `package`

Reply:
Yeap makes sense. We have to discuss it.